suite-native/module-home/src/screens/HomeScreen/components/BiometricsBottomSheet.tsx

```diff
@@ -43,24 +46,29 @@ export const BiometricsBottomSheet = () => {
     const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
     const { toggleBiometricsOption } = useBiometricsSettings();
     const isDeviceAuthorized = useSelector(selectIsDeviceAuthorized);
+
+    const isPortfolioTracker = useSelector(selectIsPortfolioTrackerDevice);
     const isCoinEnablingInitFinished = useSelector(selectIsCoinEnablingInitFinished);
     const [isCoinEnablingActive] = useFeatureFlag(FeatureFlag.IsCoinEnablingActive);
 
     const [isVisible, setIsVisible] = useState(false);
 
-    // if coin enabling is active, we need to wait for it to finish before showing the biometrics modal
-    const isCoinEnablingSetupFinished = isCoinEnablingActive ? isCoinEnablingInitFinished : true;
-
     useEffect(() => {
         let isMounted = true;
         let timerId: ReturnType<typeof setTimeout>;
         const checkBiometrics = async () => {
             const isBiometricsAvailable = await getIsBiometricsFeatureAvailable();
-            if (
-                isBiometricsAvailable &&
-                !isBiometricsOptionEnabled &&
-                isCoinEnablingSetupFinished
-            ) {
+
+            // if coin enabling is active, we need to wait for it to finish before showing the biometrics modal
+            const isCoinEnablingSetupFinished = isCoinEnablingActive
+                ? isCoinEnablingInitFinished
+                : true;
+
+            // if device is authorized, it is ready only if coin enabling setup was finished. otherwise wait
```

Comment:
The comment is confusing. Especially the "otherwise wait" part. I guess that if device is not connected (authorized), there is no wait and biometrics bottom sheet popup right away, right?

Reply:
yes, the wait is for authorized device in case coin enabling was not finished. I'll update the comment

Reply:
other than that, it looks good to me

Reply:
8820d12446